ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1342 @@
+{
```

Comment:
<p>"each imported entity <em>must</em> have an ID on it, besides features; also, each entity can <em>optionally</em> have an timestamp on it" -&gt; "each imported entity <em>must</em> have an ID; also, each entity can <em>optionally</em> have a timestamp"</p><p>"located at this public-available <a href="https://pantheon.corp.google.com/storage/browser/cloud-samples-data/ai-platform-unified/datasets/featurestore;tab=objects?project=storage-samples&amp;prefix=&amp;forceOnObjectsSortingFiltering=false" target="_blank">bucket</a>." -&gt; "located at this publicly available <a href="https://pantheon.corp.google.com/storage/browser/cloud-samples-data/ai-platform-unified/datasets/featurestore;tab=objects?project=storage-samples&amp;prefix=&amp;forceOnObjectsSortingFiltering=false" target="_blank">bucket</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>